docs/topics/serialization.md

```diff
@@ -135,14 +139,15 @@ fun main() {
 
 As a result, you get a string containing the state of this object in the JSON format: `{"a": 42, "b": "str"}`
 
-You can also serialize object collections, such as lists, in a single call.
-
-```kotlin
-val dataList = listOf(Data(42, "str"), Data(12, "test"))
-val jsonList = Json.encodeToString(dataList)
-```
+> You can also serialize object collections, such as lists, in a single call.
```

Comment:
```suggestion
> You can also serialize object collections, such as lists, in a single call:
```
Please always add colons after a sentence that you're about to demonstrate with a code sample 😄